changelog_entry.yaml

```diff
@@ -0,0 +1,5 @@
+Change log 
+- bump: minor
+  changes:
+  - Revised home mortgage interest deduction limits for pre-2017 debt.
+  - Updated references. 
```

Comment:
```suggestion
- bump: minor
  changes:
  - Revised home mortgage interest deduction limits for pre-2017 debt.
```

Reply:
the changelog entry is only updated once with the high level explanation of what the PR is aimed to achieve